Cartfile

```diff
@@ -1,5 +1,5 @@
 binary "https://customers.pspdfkit.com/carthage/mq7ic2x5FCWWUCuxWct2g87YfR9hpK/pspdfkit.json" == 8.3.1
 github "ReactiveCocoa/ReactiveSwift" ~> 6.1.0
-github "ReactiveCocoa/ReactiveCocoa" ~> 10.0
+github "ReactiveCocoa/ReactiveCocoa" "master"
```

Comment:
Are there any dangers with pinning to master? For example we don't do a carthage update for a while, but the builds always do it and manage to build a broken app because it downloads a newer master than ours? Not sure if in reality this would happen but seems like a possibility.

Reply:
Can we pin to a specific commit for now?